src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1744,6 +1745,87 @@ class Simplex_tree {
     }
   }
 
+ public:
+  /** \brief Serialize the Simplex tree - Flatten it in a vector of char */
```

Comment:
If this is a documented function, we need to say clearly that the dump is not portable, it is meant to be read in a Simplex_tree with the same options, on a computer with the same architecture.

Reply:
Add warnings in doc : [6be2663](https://github.com/GUDHI/gudhi-devel/pull/810/commits/6be26634b97f09ed6ab6c667d1ee352ce5c33e94)